.travis.yml

```diff
@@ -2,3 +2,9 @@ language: ruby
 rvm:
   - 2.1.0
   - 2.2.0
+  - 2.0.0
+  - 1.9.3
```

Comment:
[`Resqutils::StaleWorkersKiller#initialize` uses a Ruby 2 keyword argument](https://github.com/stitchfix/resqutils/blob/master/lib/resqutils/stale_workers_killer.rb#L9), so the Ruby 1.9.3 build fails for [RSpec 3](https://travis-ci.org/stitchfix/resqutils/jobs/90389794) and [RSpec 2](https://travis-ci.org/stitchfix/resqutils/jobs/90389795).

Should we refactor the method or remove 1.9.3 from CI?